shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/datasource/props/DataSourceProperties.java

```diff
@@ -119,4 +122,84 @@ public int hashCode() {
         }
         return Objects.hashCode(dataSourceClassName, stringBuilder.toString());
     }
+
+    /**
+     * can data source be aggregate.
+     *
+     * @param dataSourceProperties data source properties
+     * @throws InvalidDataSourcePropertiesException invalid data source properties exception
+     */
+    public void dataSourceCanBeAggregate(final DataSourceProperties dataSourceProperties) throws InvalidDataSourcePropertiesException {
+        if (!parseJdbcUrl().contains("jdbc:mysql")) {
+            return;
+        }
+        for (Entry<String, Object> entry : getAllLocalProperties().entrySet()) {
+            if (!dataSourceProperties.getAllLocalProperties().containsKey(entry.getKey())) {
+                String msg = "target data source item:" + entry.getKey() + ", not in source data source:" + dataSourceProperties.getAllLocalProperties();
```

Comment:
":" should be keep a space with after word

Reply:
Fix in https://github.com/apache/shardingsphere/pull/16632